install_sct

```diff
@@ -365,16 +382,19 @@ done
 # Prepare installation
 # ----------------------------------------------------------------------------------------------------------------------
 
+fetch_os_type
+check_requirements
+
 # Catch SCT version
 if [[ -e "spinalcordtoolbox/version.txt" ]]; then
-  SCT_VERSION=$(cat spinalcordtoolbox/version.txt)
+  SCT_VERSION="$(cat spinalcordtoolbox/version.txt)"
 else
   die "ERROR: version.txt not found. \n
 The install_sct script must be executed from the source directory"
 fi
 
 # Get installation type (from git or from package)
-if [[ "x$SCT_INSTALL_TYPE" == "x" ]]; then
+if [[ -z "${SCT_INSTALL_TYPE:-}" ]]; then
```

Comment:
For consistency this one should NOT use `:-`, because it's one of our internal vars and we should be able to initialize it ourselves.

Reply:
Also this should be exposed as another option flag.